tap_github/streams.py

```diff
@@ -207,6 +207,7 @@ def sync_endpoint(self,
         """
 
         # build full url
+        self.url = client.base_url
```

Comment:
It seems meaning less assignment, we can pass base_url variable in build_url function.

Reply:
Removed `url` from stream class and passed client.base_url directly instead of `self.url`.